x-pack/test/encrypted_saved_objects_api_integration/fixtures/es_archiver/key_rotation/data.json

```diff
@@ -0,0 +1,186 @@
+{
```

Comment:
__note:__ Essentially I have 6 Saved Objects here: 2 x "single-namespace",  2 x "multiple-namespaces",  2 x "hidden". Three of them are encrypted with `'a'.repeat(32)` encryption key and three with `'b'.repeat(32)`.